testsuite/tests/warnings/deprecated_module_use.compilers.reference

```diff
@@ -6,18 +6,22 @@ Alert deprecated: module Deprecated_module
   As you could guess, Deprecated_module is deprecated.
   Please use something else!
 
+
```

Comment:
The second blank line here comes for the deprecation message, which suggest that this PR might slightly affect the design of deprecation message.